incubator-tools/schema_converter_tool/schema_converter_tool.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>Can this be edited to provide more details about what "old type" and "new type" are? Provide public documentation links if possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Updated Notebook as per comments